src/core.js

```diff
@@ -49,8 +49,13 @@ var PROGRAM_CONSTRUCTORS = {
 // Creates a single application object asynchronously
 // with a gl context, a camera, a program, a scene, and an event system.
 export async function PhiloGL(canvasId, opt = {}) {
+  // ** We can't merge DEFAULT_OPTS and opt in this way; it's not a
+  // deep merge. For example, if opt.camera doesn't define fov, then
+  // the result won't have camera.fov defined. **
+  // opt = {...DEFAULT_OPTS, ...opt};
+
   // rye: TODO- use lodash.defaultsDeep instead of $merge.
```

Comment:
I see. I definitely want to get rid of the jquery emulation code, especially as a lot of it is covered by ES6 constructs.

But for simplicity of build etc, I also want to limit (eliminate) dependencies on other libraries, like lodash. 

If we do need a deep merge function, we could just rename the current function from $.merge to deepMerge, add some JSDoc to it, and put it in a utils.js file.